### ISSUE TITLE
2 - Reconsider iOS7 multiple attribute workaround

### ISSUE THREAD

**Issue by feltnerm** [FineUploader/fine-uploader#1184]
If the `validation.allowedExtensions` option is left empty, regardless of what `validation.acceptFiles` is set to, then iOS7 users are unable to select multiple files even if the `multiple` option is enabled.

In #1039 we added an internal workaround for an iOS7 issue with .mov files that would

> "... omit the file input element's multiple attribute if the validation.allowedExtensions value is any empty array, or if it contains an entry for '.mov'" files AND the current OS is iOS 7." 

I think this is the cause of the issue.

We may need to rethink the case when `validation.allowedExtensions` is the empty array. Integrators the _must_ specify a whitelist of validation extensions when supporting iOS7 and multiple file selection. On the other hand, .mov files would cause errors by default.

It seems pretty unusual to include values for the acceptFiles option and not allowedExtensions.  Of the two, allowedExtensions is the only reliable one.  acceptFiles is just a proxy for the accept attribute on the file input element.  This attribute has very poor cross browser support and functions differently between the browsers that do support it.  

In short, the allowedExtensions option shouldn't normally be empty if acceptFiles values have been provided.

---

Solutions:
1. If the `validation.acceptFiles` option or `validation.allowedExtensions` option disallows the uploading of .mov files (i.e., no "video/*" or "video/quicktime" in acceptFiles, or no "mov" in allowedExtensions), then multiple file selection should be allowed to be enabled by the integrator
2. Keep it the way it is and educate integrators on how to enable the multiple option on iOS7.
